features/networking/operator.feature

```diff
@@ -345,3 +345,25 @@ Feature: Operator related networking scenarios
     And all existing pods die with labels:
       | app=dhcp-daemon |
     """
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-27333
+  @admin
+  @destructive
+  Scenario: Changing mtu in CNO should not be allowed
+    Given the mtu value "1750" is patched in CNO config according to the networkType
+    And admin uses the "openshift-network-operator" project
+    When I run the :get admin command with:
+      | resource | pods                               |
+      | o        | jsonpath={.items[*].metadata.name} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :network_operator_pod clipboard
```

Comment:
What will happen if there are multiple pods in the openshift-network-operator project ?
How about use `Given a pod becomes ready with labels:` ?

Reply:
I guess its okay to leave as is @liangxia. ```Openshift-network-operator``` system project is expected to have only one pod in it. We can definitely take ```Given a pod becomes ready with labels:``` but that needs to be accompanied by pseudo admin user step. Let me know WDYT

Reply:
@liangxia ?